instana/w3c_trace_context/treceparent.py

```diff
@@ -0,0 +1,64 @@
+# (c) Copyright IBM Corp. 2021
+# (c) Copyright Instana Inc. 2021
+
+from ..log import logger
+import re
+
+
+class Traceparent:
+    SPECIFICATION_VERSION = "00"
+    TRACEPARENT_REGEX = re.compile("[0-9a-f]{2}-(?!0{32})([0-9a-f]{32})-(?!0{16})([0-9a-f]{16})-[0-9a-f]{2}")
```

Comment:
```suggestion
    TRACEPARENT_REGEX = re.compile("^[0-9a-f]{2}-(?!0{32})([0-9a-f]{32})-(?!0{16})([0-9a-f]{16})-[0-9a-f]{2}")
```
I believe this should make the regex compliant with future versions as well